source/documentation/tools/create-a-derived-table/quick-reference.md

```diff
@@ -62,6 +68,90 @@ This list comprises everything you need to do and consider to get set up and rea
+
+<li>Check the status of the repository with <code>git status</code>; if it's not on the main branch, switch to it using <code>git checkout main</code>.</li>
+
+<li>Pull the latest content from the remote repository and update your local repository immediately with <code>git pull</code>.</li>
+
+<li>To create your own branch, use the command <code>git checkout -b project_name</code>.
+<li>Add changes from the working directory to the staging area with <code>git add file_name</code>.</li>
+<li>Save your changes to the local repository by using <code>git commit -m "messages"</code>.</li>
+
+<li>To see the changes that have occurred in the remote branch, perform a <code>git fetch</code>.</li>
+
+<li>If there are any changes in the remote repository, update your local branch by executing <code>git pull</code>.</li>
+
+<li>Switch to your own branch with the command <code>git switch your_branch_name</code>.</li>
+
+<li>Update the main branch by merging your changes into it using <code>git merge main -m "messages"</code>.</li>
+</ul>
+
+## Important dbt commands
+<ul>
+    <li><code>dbt clean</code> to remove run artefacts from previous invocations of dbt</li>
+    <li><code>dbt compile --select models/.../path/to/my/models/</code>, to check your SQL and YAML is syntactically correct</li>
+    <li><code>dbt run --select models/.../path/to/my/models/</code>, to deploy your models</li>
+    <li><code>dbt seed --select seeds/.../path/to/my/seeds/</code>, to deploy your seeds</li>
+    <li><code>dbt test --select models/.../path/to/my/models/</code>, to run tests on models with tests defined</li>
+    <li><code>dbt docs generate</code>, to generating your project's documentation </li>
+    <li><code>dbt docs serve </code>, to serve your documentation locally </li>
+</ul> 
+
+## yamllint commands
+Linting is the automated checking of your code for programmatic and stylistic errors performed by running a &lsquo;linter&rsquo;
+<ul>
+    <li><code>yamllint .../path/to/yaml/file.yaml</code>, to lint a single YAML file</li>
+    <li><code>yamllint .../path/to/yaml/directory/</code>, to lint a whole directory of YAML files</li>
+</ul>
+
+## sqlfluff commands
+- To format SQL files using SQLFluff
+<ul>
+    <li><code>sqlfluff lint .../path/to/sql/file.sql</code>, to lint a single SQL file</li>
+    <li><code>yamllint .../path/to/yaml/directory/</code>, to lint a whole directory of SQL files</li>
+</ul>
+
+##  Moving the model to Prod:
+If you're planning to submit a pull request for your models and subsequently merge them into the main repository, follow this recommended checklist:
+
+<ol type="1">
+  <li><b>Successful Deployment in Development: </b>Ensure that the development models successfully pass through the deploy-dev workflow. This step is crucial to guarantee that the model complies with linting and undergoes other essential checks.</li>
```

Comment:
May want to distinguish between deploying from local IDE and via Git PR. Ideally you'd want to make sure your models are working and linted locally before committing the changes and raising the PR (in which the deploy-dev is then a final check).
Also make sure the branch is up to date with main and any conflicts resolved.

Reply:
good point; want to ensure people lint and run tests locally before raising the PR. The checklist should be in chronological/process order too - will make it easier to follow.